pontoon/homepage/templates/homepage_content.html

```diff
@@ -128,7 +128,7 @@ <h2>One more thing.</h2>
           making you the first proofreader and tester.
           Talk about context?
         </p>
-        <a class="button primary" href="{{ intro_url }}">Try In-context Localization</a>
+        <a class="button primary" href="{% url 'pontoon.translate.locale.agnostic' 'pontoon-intro' 'messages.properties' %}">Try In-context Localization</a>
```

Comment:
As I thought, this is django template syntax. I guess it's fine to use that instead of jinja2. :-)